src/main/java/org/opensearch/neuralsearch/processor/factory/NormalizationProcessorFactory.java

```diff
@@ -50,8 +51,9 @@ public SearchPhaseResultsProcessor create(
     ) throws Exception {
         Map<String, Object> normalizationClause = readOptionalMap(NormalizationProcessor.TYPE, tag, config, NORMALIZATION_CLAUSE);
         ScoreNormalizationTechnique normalizationTechnique = ScoreNormalizationFactory.DEFAULT_METHOD;
+        String normalizationTechniqueName = MinMaxScoreNormalizationTechnique.TECHNIQUE_NAME;
```

Comment:
Why we take a default value here?

Reply:
We assigned a default value but with the other logic change. Reverted this.